app/Http/Controllers/PollController.php

```diff
@@ -34,4 +35,31 @@ public function create(Request $request): JsonResponse
         $poll->save();
         return response()->json($poll, status: 201);
     }
+
+    public function getAll(): JsonResponse
+    {
+        $polls = Poll::all();
+        $convertedPolls = [];
+
+        foreach ($polls as $poll) {
+            $votes = Vote::where('poll_id', $poll->id)->get();
+            $upvotes = $votes->where('description', true)->count();
+            $downvotes = $votes->where('description', false)->count();
+
+            $convertedPoll = [
+                "id" => $poll->id,
+                "name" => $poll->name,
+                "description" => $poll->description,
+                "endDate" => $poll->end_date,
```

Comment:
to be consistent here as api, i would recommend snake case like `updated_at` and so on.